### PR TITLE
HM about Bob and Sarah

### PR DESCRIPTION
**Pull request by MatveiNikita**

### CODE REVIEW

File: Homework№10/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    } else if (bmi > 18.5 && bmi < 25) {
+      return "You have normal wieght";
+    } else {
+      return "You're B.I.G";
+    }
+  },
+};
+
+console.log(Bob.checkIBM())
+console.log(Sarah.checkIBM());
```
Comment (IvanCeban) [inline Homework№10/script.js:38]: WOW!
 Nice solution!
 
 But it seems like you're repeating yourself. Creating the same functions twice. Can you optimize it?

**Review by IvanCeban — CHANGES_REQUESTED** [review]

File: Homework№10/script.js (hunk 1 above)
Comment (MatveiNikita) [reply Homework№10/script.js:38]: Do you mean that I can call both methods in one method console.log?

Comment (MatveiNikita) [reply Homework№10/script.js]: Please look at my changes
Thank you for yuor feedback

File: Homework№10/script.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    } else if (bmi > 18.5 && bmi < 25) {
+      return "You have normal wieght";
+    } else {
+      return "You're B.I.G";
+    }
+  },
+};
+
+console.log(Bob.checkIBM(), Sarah.checkIBM())
+
```
Comment (IvanCebanPVT) [inline Homework№10/script.js:38]: I mean you're duplicating the method checkIBM for both Bob and Sarah.

You can create a separate function and assign it to checkIBM key of Bob and Sarah. 

This is called DRY - Don't Repeat Yourself.

If the code is duplicated - place it into a variable (or in your case - function)

**Review by IvanCebanPVT — CHANGES_REQUESTED** [review]

**Review by IvanCeban — APPROVED** [review]
Now it's great!

**Pull request merged**